### PR TITLE
Remove aspect ratio from `YouTubeBlock`

### PR DESCRIPTION
**Pull request by SebiVPS**
Due to the media block standardisation and following from this the video block reworkings we decided to remove the aspectRatio from the YouTubeBlock

### CODE REVIEW

**Comment by johnnyomair** [comment]
Can you confirm that this doesn't need a block migration?

**Comment by SebiVPS** [comment]
yes, in my tests it looked ok

File: packages/api/blocks-api/src/blocks/YouTubeVideoBlock/1-remove-aspect-ratio.migration.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/* eslint-disable @typescript-eslint/naming-convention */
+enum AspectRatio {
+    "16X9" = "16X9",
+    "4X3" = "4X3",
+}
+/* eslint-enable @typescript-eslint/naming-convention */
+
+interface From {
+    youtubeIdentifier?: string;
+    aspectRatio: AspectRatio;
```
Comment (johnnyomair) [inline packages/api/blocks-api/src/blocks/YouTubeVideoBlock/1-remove-aspect-ratio.migration.ts:13]: Simply use any here 😁

File: packages/api/blocks-api/src/blocks/YouTubeVideoBlock/youtube-video.block.ts
Comment (johnnyomair) [inline packages/api/blocks-api/src/blocks/YouTubeVideoBlock/youtube-video.block.ts:1]: Please move this into a `migrations/` subfolder

File: packages/site/cms-site/src/blocks/YouTubeVideoBlock.tsx [hunk 2]
```diff
@@ -69,10 +43,29 @@ export const YouTubeVideoBlock = withPreview(
         youtubeUrl.search = searchParams.toString();
 
         return (
-            <VideoContainer $heightInPercent={getHeightInPercentForAspectRatio(aspectRatio)}>
-                <iframe src={youtubeUrl.toString()} style={{ border: 0 }} />
+            <VideoContainer $aspectRatio={aspectRatio?.replace("x", " / ")}>
```
Comment (johnnyomair) [inline packages/site/cms-site/src/blocks/YouTubeVideoBlock.tsx:46]:
```suggestion
            <VideoContainer $aspectRatio={aspectRatio.replace("x", " / ")}>
```

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**